File: bronze/28702/c++/main.cpp
#include <iostream>
#include <string>

int main()
{
    std::string num[3];
    int number = 0;
    std::cin >> num[0] >> num[1] >> num[2];

    for (int i = 0; i < 3; i++) {
        number++;
        if (num[i] != "FizzBuzz" && num[i] != "Fizz" && num[i] != "Buzz") {
            number = std::stoi(num[i]);
        }
    }
    number++;
    if (number % 3 == 0 && number % 5 == 0) {
        std::cout << "FizzBuzz\n";
    } else if (number % 3 == 0 && number % 5 != 0) {
        std::cout << "Fizz\n";
    } else if (number % 3 != 0 && number % 5 == 0) {
        std::cout << "Buzz\n";
    } else {
        std::cout << number;
    }

    return 0;
}

File: bronze/28702/python/main.py

if (__name__ == "__main__"):
    num = list(input() for _ in range(3))
    number = 0

    for i in range(3):
        try:
            number = int(num[i])
        except:
            number += 1
    number += 1
    
    if (number % 3 == 0 and number % 5 == 0):
        print("FizzBuzz")
    elif (number % 3 == 0 and number % 5 != 0):
        print("Fizz")
    elif (number % 3 != 0 and number % 5 == 0):
        print("Buzz")
    else:
        print(number)


